src/legacy/promise/actionWrappers.ts

```diff
@@ -1,4 +1,4 @@
-import action from '../action';
+import { action } from 'satcheljs';
```

Comment:
I don't think this is right. This will be pulling in the new v3 action, instead of the legacy action.